recipes/DABEST/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "dabest" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8147e304ebc587923f0e2692ff68eb33e5abba81d0b93994da1d14cc4479dec4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - matplotlib
+    - seaborn
+    - pandas
+    - numpy
+    - scipy
+    - pytest
+
+test:
+  imports:
+    - pytest
+    - sys
+    - numpy
+    - scipy
+    - pandas
+    - seaborn
+
+about:
+  home: https://acclab.github.io/DABEST-python-docs/index.html
+  license: 'BSD 3-clause'
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
The license should probably be packaged with the source distribution. For now, you could include the following link in the recipe as a comment:

https://github.com/ACCLAB/DABEST-python/issues/12

Reply:
Done, thanks for the heads up!

Reply:
It looks like the `0.1.1` tarball now has the license, so can you delete the license from the recipe directory and change this line to `license_file: LICENSE`. 

After that, everything looks good to merge.

Reply:
pinging this!

Reply:
sorry, was out of the office, done!